### ISSUE TITLE
Types: Property 'title' does not exist on type 'SVGElement'.

### ISSUE THREAD

**Issue by fregante** [proteriax/jsx-dom#46]
It seems that intrinsic elements are not automatically correctly typed:
```tsx
const container = <div/>;
container.title
//        ~~~~~
// Property 'title' does not exist on type 'ReactElement'.
//   Property 'title' does not exist on type 'SVGElement'.
```

It seems that every such use must be used as:

```tsx
const container = <div/> as HTMLElement;
container.title
```

Using [dom-chef](https://github.com/vadimdemedes/dom-chef/tree/v4.2.2) with [some custom types](https://github.com/sindresorhus/refined-github/blob/41011ebabfde9fc1b44ab27fb321a04d5c0c13bd/source/globals.d.ts#L36-L54), [this has been possible in Refined GitHub](https://github.com/sindresorhus/refined-github/blob/41011ebabfde9fc1b44ab27fb321a04d5c0c13bd/source/features/user-local-time.tsx#L132). However I think this came with some drawbacks as well.

**Comment by fregante** [comment]
Question was: Do you think the assertion could be avoided by adjusting the types?

**Comment by proteriax** [comment]
The reason the intrinsic elements are not typed is documented in README.md Known Issues section:

> `<div />`, and other tags, are inferred as a general JSX.Element in TypeScript instead of HTMLDivElement (or the equivalent types). This is a known bug and its fix depends on TypeScript#21699.

The approach dom-chef takes, if I’m understanding correctly, is to make all JSX element returnees inherit from HTMLDivElement, which is definitely not something I want to see happen.
